src/base-command.ts

```diff
@@ -72,6 +72,9 @@ export default abstract class BaseCommand extends Command {
   }
 
   async finally(_: Error | undefined): Promise<any> {
```

Comment:
We are also not calling super.finally which we probably should do. See:
https://github.com/oclif/core/blob/main/src/command.ts#L268

Anytime we override a base class method you almost always want to call the super method.

Reply:
super.finally() and super.catch() should be executed at the end of the endSentryTransaction() function.

https://github.com/architect-team/architect-cli/blob/rc/src/base-command.ts#L189-L192